proxies/live/apiproxy/targets/sync-wrap.xml

```diff
@@ -33,6 +33,18 @@
         <Condition>(response.status.code Equals "504")</Condition>
         <Name>Quota</Name>
       </Step>
+      <Step>
+        <Condition>(response.status.code Equals "429")</Condition>
+         <Name>AssignMessage.Errors.ServiceUnavailable</Name>
+      </Step>
+      <Step>
+        <Condition>(response.status.code Equals "401")</Condition>
+         <Name>AssignMessage.Errors.ServiceUnavailable</Name>
+      </Step>
+      <Step>
```

Comment:
I think this step is unnecessary - also it won't ever get triggered (see the faultrule condition)

Reply:
Should it have a comment also saying it's a no-op?

Reply:
The 503 condition was included because if my memory is correct, when it would hit a 401 or a 429 and run `AssignMessage.Errors.ServiceUnavailable`, I believe there were some formatting errors which would result in a Frankensteins monster of a 401/429 and a 503. So once it has completed ServiceUnavailable, it would then check to see if it had been assigned to a 503 and if so, fill in the FHIR formatted response in `AssignMessage.Errors.CatchAllMessage.`

I could be wrong though as I've slept since then.

Reply:
![This is the response for a 429 without this step.](https://user-images.githubusercontent.com/69810386/121387181-64236d80-c942-11eb-98e3-51cfab6a5fa3.png)

Reply:
Above is the response if this step is removed.

Reply:
![image](https://user-images.githubusercontent.com/69810386/121387658-c2e8e700-c942-11eb-8a77-efe35a4cc85b.png)

Reply:
Above is the response including this step.

Reply:
Shouldn't it be in it's own fault rule? It's within the generic pass through fault rule atm that's used to pass-through valid spine errors to the client. I would copy the request_timeout fault rule and add conditions in the pass through to ignore 401/429